### PR TITLE
Adding brand score image in the pdp

### PR DESCRIPTION
**Pull request by AvishekPrasad**
Adding brand score image in the pdp

### CODE REVIEW

File: snippets/product-main-info.liquid [hunk 1]
```diff
@@ -1,3 +1,14 @@
+<style>
+  .brand_score_img_cus{
+    height: 75px;
```
Comment (san-mobi) [inline snippets/product-main-info.liquid:3]: @AvishekPrasad  - Height should not be fixed.It should be WIDTH & then HEIGHT AUTO. 
Image that we will put would be Square.

File: snippets/product-main-info.liquid [hunk 2]
```diff
@@ -131,12 +142,12 @@
...
       {%comment%}<span class="product-form__info-title text--strong">{{ 'product.form.price' | t }}:</span>{%endcomment%}
 
-      <div class="product-form__info-content" role="region" aria-live="polite">
+      <div class="product-form__info-content display_offer_image" role="region" aria-live="polite">
         <div class="price-list actual-product-price">
           {%- if selected_variant.compare_at_price > selected_variant.price -%}
           <span class="price price--highlight">
             <span class="visually-hidden">{{ 'product.general.sale_price' | t }}</span>
-            {{- selected_variant.price | money -}}  &nbsp;<small style="color:#777; font-size:18px;display:inline-block;margin-bottom:11px;">MRP </small>
+            <small style="color:#9a7058; font-size:23px;display:inline-block;margin-bottom:2px;">MRP: </small> &nbsp; {{- selected_variant.price | money -}}
```
Comment (san-mobi) [inline snippets/product-main-info.liquid:150]: CSS in style tags must be avoided. @AvishekPrasad

Comment (AvishekPrasad) [reply snippets/product-main-info.liquid:150]: Sir It was already present so I have'nt changed it , just modified the css that was required

File: snippets/product-main-info.liquid (hunk 1 above)
Comment (AvishekPrasad) [reply snippets/product-main-info.liquid:3]: done sir

**Pull request merged**